src/modules/sdlog2/sdlog2_messages.h

```diff
@@ -185,8 +185,20 @@ struct log_STAT_s {
 	float load;
 };
 
+/* --- CONTROL STATE --- */
+#define LOG_CTS_MSG 11
```

Comment:
The last reordering introduced a bug, and I see 46 missing here. Please just add the message to the end of the list and please do not modify everything to just add a new message. Its very hard to maintain and if every contributor takes this approach results in lots of merge conflicts.